packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -0,0 +1,110 @@
+import { svg, css, LitElement } from '@lion/core';
+import { localize, LocalizeMixin } from '@lion/localize';
+
+export class LionProgressIndicator extends LocalizeMixin(LitElement) {
+  static get styles() {
+    return [
+      css`
+        :host {
+          display: inline-block;
+        }
+
+        svg.lion-progress {
+          animation: progress-indicator-rotate 2s linear infinite;
+          height: 40px;
+          width: 40px;
+        }
+
+        @keyframes progress-indicator-rotate {
+          to {
+            transform: rotate(360deg);
+          }
+        }
+      `,
+    ];
+  }
+
+  static get localizeNamespaces() {
+    return [
+      {
+        'lion-loading-indicator': locale => {
+          switch (locale) {
+            case 'bg-BG':
+            case 'bg':
+              return import('../translations/bg.js');
+            case 'cs-CZ':
+            case 'cs':
+              return import('../translations/cs.js');
+            case 'de-DE':
+            case 'de':
+              return import('../translations/de.js');
+            case 'en-AU':
+            case 'en-GB':
+            case 'en-US':
+            case 'en-PH':
+            case 'en':
+              return import('../translations/en.js');
+            case 'es-ES':
+            case 'es':
+              return import('../translations/es.js');
+            case 'fr-BE':
+            case 'fr-FR':
+            case 'fr':
+              return import('../translations/fr.js');
+            case 'hu-HU':
+            case 'hu':
+              return import('../translations/hu.js');
+            case 'it-IT':
+            case 'it':
+              return import('../translations/it.js');
+            case 'nl-BE':
+            case 'nl-NL':
+            case 'nl':
+              return import('../translations/nl.js');
+            case 'pl-PL':
+            case 'pl':
+              return import('../translations/pl.js');
+            case 'ro-RO':
+            case 'ro':
+              return import('../translations/ro.js');
+            case 'ru-RU':
+            case 'ru':
+              return import('../translations/ru.js');
+            case 'sk-SK':
+            case 'sk':
+              return import('./translations/sk.js');
+            case 'uk-UA':
+            case 'uk':
+              return import('../translations/uk.js');
+            default:
+              return import('../translations/en.js');
+          }
+        },
+      },
+    ];
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  template() {
```

Comment:
This and the styles are more or less superfluous as they will always be overridden. I don't feel like they add much value, unless someone intends to use Lion directly without extending the components, which I think is not a use case, right?

Reply:
maybe call this `_graphicTemplate` ? (to comply with our naming conventions)

Reply:
I will change this